### DISCUSSION TITLE
👀ARM and v0.81

### DISCUSSION THREAD

**Discussion by lin-ycv** [lin-ycv/EverythingPowerToys#95] [Announcements]
ARM support was requested quite a while back, but I don't have an ARM device to test ARM compatibilty, so I've never dealt with it properly.

Between seeing ARM users showing up in the issues from time to time, and the pending release of Snapdragon X Elite. I've decided to support ARM properly starting v0.81.

Previous releases technically supported ARM, but required users to manually swap out Everything64.dll for an ARM version. In version 0.81.0, both the ARM and x86 version of the dll will be included, on launch, the plugin will check which architecture your CPU is on, and use the correct one while deleting the unnecessary one.

Hopefully this will make things easier for ARM users.

**Comment by dXrayb** [comment]
Bravo👍
